sanic/mixins/routes.py

```diff
@@ -790,24 +790,9 @@ def _generate_name(self, *objects) -> str:
 
         return name
 
-    async def _static_request_handler(
-        self,
-        file_or_directory,
-        use_modified_since,
-        use_content_range,
-        stream_large_files,
-        request,
-        content_type=None,
-        __file_uri__=None,
-    ):
-        # Merge served directory and requested file if provided
+    async def get_file_path(self, file_or_directory, __file_uri__, not_found):
```

Comment:
The dunder seems odd here. I see it is in the handler args, but still seems out of place. Maybe out of scope for this PR to change.